gslib/tests/test_signurl.py

```diff
@@ -125,6 +125,16 @@ def testSignUrlOutputJSON(self):
     self.assertIn('%2Fus-central1%2F', stdout)
     self.assertIn('\tPUT\t', stdout)
 
+  def testSignUrlWithJSONKeyFileAndObjectGeneration(self):
+    """Tests signurl output of a sample object version with json keystore."""
+    bucket_uri = self.CreateBucket(versioning_enabled=True)
+    object_uri = self.CreateObject(bucket_uri=bucket_uri, contents=b'z')
+    cmd = ['signurl', self._GetJSONKsFile(), object_uri.version_specific_uri]
+    stdout = self.RunGsUtil(cmd, return_stdout=True)
+    self.assertIn('x-goog-credential=test%40developer.gserviceaccount.com',
```

Comment:
Is there a constant for this somewhere?
I see other tests are typing out the string but that seems wrong

Reply:
There wasn't one -- it comes from a file somewhere. I added one though.